src/content/docs/features/ai-toolbox.md

```diff
@@ -55,5 +55,21 @@ To break it down, you need to provide the following information:
 Once you finish, save the file, and your custom toolbox command will be available.
 When entering the `/help` command, you will see your custom command in the list of available commands.
 
+![Refact Toolbox](../../../assets/custom_command.png)
 
-![Refact Toolbox](../../../assets/custom_command.png)
\ No newline at end of file
+## Customizable System Prompt
```

Comment:
first we need to tell about default system prompt and how it works with chat and toolbox
also we need to add the reason of this feature and real use cases
for example you're java developer and all generated code should be in java lang: just add default system prompt that model should generate only java code